File: TopCoder02/Java/Main.java
public class Main {

	public static void main(String[] args) {
		String[] first = {"fishing", "gardening", "swimming", "fishing"};
		String[] second = {"hunting", "fishing", "fishing", "biting"};

		InterestingParty ip = new InterestingParty();

		System.out.println(ip.bersInvitation(first, second));
	}

}

File: TopCoder02/Scala/Main.scala
object Main {

  def main(args: Array[String]): Unit = {
    val first = Array("fishing", "gardening", "swimming", "fishing")
    val second = Array("hunting", "fishing", "fishing", "biting")
    val ip = new InterestingParty
    println(ip.bestInvitation(first, second))
    
  }

}